src/view/com/pager/PagerWithHeader.tsx

```diff
@@ -155,15 +150,7 @@ export const PagerWithHeader = React.forwardRef<PagerRef, PagerWithHeaderProps>(
       if (!throttleTimeout.current) {
         throttleTimeout.current = setTimeout(() => {
           throttleTimeout.current = null
-
           runOnUI(adjustScrollForOtherPages)()
-
-          const nextIsScrolledDown = scrollY.value > SCROLLED_DOWN_LIMIT
-          if (isScrolledDown !== nextIsScrolledDown) {
-            React.startTransition(() => {
-              setIsScrolledDown(nextIsScrolledDown)
```

Comment:
Pulling this down to the `List` improves perf here too.

Scrolling profile down before:

<img width="645" alt="Screenshot 2023-12-13 at 06 23 31" src="https://github.com/bluesky-social/social-app/assets/810438/bd539274-8e20-4f6f-a3d4-57b0d097f22e">

Scrolling profile down after:

<img width="641" alt="Screenshot 2023-12-13 at 06 24 43" src="https://github.com/bluesky-social/social-app/assets/810438/e5371d7d-ff81-4a2c-8bc1-e4fa5cda0081">